include/bluetooth/mesh/light_hsl_cli.h

```diff
@@ -0,0 +1,463 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/**
+ *  @file
+ *  @defgroup bt_mesh_light_hsl_cli Light HSL Client model
+ *  @{
+ *  @brief API for the Light HSL Client model.
+ */
+
+#ifndef BT_MESH_LIGHT_HSL_CLI_H__
+#define BT_MESH_LIGHT_HSL_CLI_H__
+
+#include <bluetooth/mesh/light_hsl.h>
+#include <bluetooth/mesh/model_types.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+struct bt_mesh_light_hsl_cli;
+
+/** @def BT_MESH_MODEL_LIGHT_HSL_CLI
+ *
+ *  @brief Light HSL Client model composition data entry.
+ *
+ *  @param[in] _cli Pointer to a @ref bt_mesh_light_hsl_cli instance.
+ */
+#define BT_MESH_MODEL_LIGHT_HSL_CLI(_cli)                                      \
+	BT_MESH_MODEL_CB(BT_MESH_MODEL_ID_LIGHT_HSL_CLI,                       \
+			 _bt_mesh_light_hsl_cli_op, &(_cli)->pub,              \
+			 BT_MESH_MODEL_USER_DATA(struct bt_mesh_light_hsl_cli, \
+						 _cli),                        \
+			 &_bt_mesh_light_hsl_cli_cb)
+
+/** Light HSL Client state access handlers. */
+struct bt_mesh_light_hsl_cli_handlers {
+	/** @brief HSL status message handler.
+	 *
+	 *  @param[in] cli    Client that received the status message.
+	 *  @param[in] ctx    Context of the message.
+	 *  @param[in] status HSL status contained in the
+	 *                    message.
+	 */
+	void (*const hsl_status)(struct bt_mesh_light_hsl_cli *cli,
+				 struct bt_mesh_msg_ctx *ctx,
+				 const struct bt_mesh_light_hsl_status *status);
+
+	/** @brief HSL target status message handler.
+	 *
+	 *  @param[in] cli    Client that received the status message.
+	 *  @param[in] ctx    Context of the message.
+	 *  @param[in] status HSL target status contained in the
+	 *                    message.
+	 */
+	void (*const hsl_target_status)(struct bt_mesh_light_hsl_cli *cli,
+				 struct bt_mesh_msg_ctx *ctx,
+				 const struct bt_mesh_light_hsl_status *status);
+
+	/** @brief Default parameter status message handler.
+	 *
+	 *  @param[in] cli    Client that received the status message.
+	 *  @param[in] ctx    Context of the message.
+	 *  @param[in] status Default HSL value of the server.
+	 */
+	void (*const default_status)(
```

Comment:
This is strange the previous two handlers have prefix `hsl_` the others don't. Is there any reason for that?

Reply:
This is strange. I'll remove the `hsl_` prefix, as there's no reason to keep it for all of them.